src/Layout/Footer/Footer.js

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+/** Services */
+import { skipCurrentSong } from '@services/api';
 /** Utils */
 /** Components */
 import ProgressBar from './ProgressBar';
 import NowPlaying from './NowPlaying';
 /** Styled components */
 import { FooterContainer, FooterWrapper } from './footer.styled';
 
+import './SkipButton.scss';
+
 const Footer = () => {
   const current = useSelector(state => state.playing.current);
 
+  const skipHandler = async () => {
```

Comment:
You don't need to do this right now, but it would be interesting to use the `useCallback` hook here to prevent unnecessary renders.

The main idea of the `useCallback` hook is that it will return a memoized version of the callback that only changes if one of the dependencies has changed.

Reply:
Done.